server/config/config.go

```diff
@@ -338,6 +354,12 @@ func DefaultConfig() *Config {
 				Fsync: false,
 			},
 		},
+		Streaming: StreamingConfig{
+			ABCI: ABCIListenerConfig{
+				Keys:          []string{"*"},
```

Comment:
I feel like `[]string{}` would be a better default.

Reply:
It's hard to determine what a default value should be here. I settled for listening to all `*` as the default to have one less property to configure so you'd be left with setting the `plugin` property.

Reply:
I think that, by default, nothing should be streamed. If a node operator wants to enable streaming, part of that setup is updating the config anyway, and would be when they define which keys to include.

Even if the "plugin" value is empty, doesn't having `"*"` in the keys list make a bunch of listener stuff get wired up?

Reply:
> Even if the "plugin" value is empty, doesn't having "*" in the keys list make a bunch of listener stuff get wired up?

No, no listeners are attached unless a listening plugin is configured. But I'll the default an empty list.

Reply:
default  is now `[]string{}`